src/main/scala/centaur/test/Test.scala

```diff
@@ -67,7 +66,8 @@ object Operations {
         // Collect only the parameters which exist:
         val params = List("wdlSource" -> Option(workflow.data.wdl),
           "workflowInputs" -> workflow.data.inputs,
-          "workflowOptions" -> workflow.data.options) collect { case (name, Some(value)) => (name, value) }
+          "workflowOptions" -> WorkflowOptions(workflow.data.options).insertSecrets.options
```

Comment:
meh.  I would change it if I were going in here, but I don't really feel that strongly about it.